slack_bolt/cli/error/__init__.py

```diff
@@ -0,0 +1,2 @@
+class CliError(Exception):
```

Comment:
Can move this class to https://github.com/slackapi/bolt-python/blob/main/slack_bolt/error/__init__.py ?

Reply:
I've added this change, thanks